app/frontend.py

```diff
@@ -55,15 +62,20 @@ def rtsp_snapshot():
                 resp[name] = wb.rtsp_snap(name, wait=False)
         return resp
 
-    @app.route("/img/<path:path>")
-    def img(path: str):
-        """Use ffmpeg to take a snapshot from the rtsp stream."""
-        wait: bool = request.args.get('wait', default=True, type=bool)
-        uri = Path(path).stem
+    @app.route("/img/<path:img_file>")
+    def img(img_file: str):
+        """
+        Serve static image if image exists or take a new snapshot from the rtsp stream.
+
+        rtsp arg to wait for new image from rtsp stream.
+        """
+        if request.args.get("rtsp") is None and Path(wb.img_path + img_file).exists():
```

Comment:
Setting the static image to default has broken this feature. The benefit of having rtsp_snap() is no longer needing to rely on SNAPSHOT=RTSP.

I think in most cases, when people invoke /img/cam-name.jpg their expectation is they will be served an up to date image. The use case for serving up a stale image is pretty niche (and originally only because rtsp_snap() didn't exist, and we were stuck with using the SNAPSHOT=RTSP mechanism).

Reply:
You're probably right and we should default to a fresh rtsp image. 

I think we should still keep a uri to load a stale image for the web-ui (or placeholder image if we don't have a stale image) on initial load while we wait for js to wait for the rtsp image.

We can probably also get rid of the rtsp_snap uri and the related js.

We may be able to pull the alarm/photo files from the SD card from some cameras so we may want to use that as an option in the future.

Reply:
> You're probably right and we should default to a fresh rtsp image.
> 
> I think we should still keep a uri to load a stale image for the web-ui (or placeholder image if we don't have a stale image) on initial load while we wait for js to wait for the rtsp image.
> 
> We can probably also get rid of the rtsp_snap uri and the related js.
> 
> We may be able to pull the alarm/photo files from the SD card from some cameras so we may want to use that as an option in the future.

I volume mount my /img/ directory in docker, so it's persistent and I always have an image. So I've yet to actually see the loading functionality.